src/components/RangeSlider.react.js

```diff
@@ -82,8 +93,15 @@ export default class RangeSlider extends Component {
                     }}
                     tipProps={tipProps}
                     value={value}
+                    marks={truncatedMarks}
                     {...omit(
-                        ['className', 'value', 'setProps', 'updatemode'],
+                        [
+                            'className',
+                            'value',
+                            'setProps',
+                            'marks',
+                            'updatemode',
+                        ],
```

Comment:
Not for now, but just because I'm thinking about it: I'd like us to consider moving away from `omit` for forwarding props to 3rd-party components, and use `pick` instead. IMO `omit` makes it too hard to follow which props are being passed on, too easy to forget to remove newly added props, and makes us lazy about accepting someone else's choice of names and values even if they don't make sense in our case. (cc @Marc-Andre-Rivet )

Reply:
@alexcjohnson Agreed for DCC! There's also a bit of this happening in the table fragments / d3-format wrappers that should be looked into. Due to its generated nature I'd say whether this makes sense on HTML components would have to be evaluated independently. Can you open an issue for follow up and tag it for `Dash v1.x` milestone? Thanks.